sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_operations/_operations.py

```diff
@@ -122,3 +158,141 @@ def analyze_conversation(
         return deserialized
 
 
+
+    def _submit_conversation_job_initial(
+        self,
+        body: _models.AnalyzeConversationJobsInput,
+        **kwargs: Any
+    ) -> Optional[_models.AnalyzeConversationJobState]:
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
+        }
+        error_map.update(kwargs.pop('error_map', {}) or {})
+
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
+
+        api_version = kwargs.pop('api_version', _params.pop('api-version', "2022-04-01-preview"))  # type: str
+        content_type = kwargs.pop('content_type', _headers.pop('Content-Type', "application/json"))  # type: Optional[str]
+        cls = kwargs.pop('cls', None)  # type: ClsType[Optional[_models.AnalyzeConversationJobState]]
+
+        _json = self._serialize.body(body, 'AnalyzeConversationJobsInput')
+
+        request = build_submit_conversation_job_request_initial(
+            api_version=api_version,
+            content_type=content_type,
+            json=_json,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "Endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, 'str', skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)  # type: ignore
+
+        pipeline_response = self._client._pipeline.run(  # type: ignore # pylint: disable=protected-access
+            request,
+            stream=False,
+            **kwargs
+        )
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200, 202]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        deserialized = None
+        response_headers = {}
+        if response.status_code == 200:
+            deserialized = self._deserialize('AnalyzeConversationJobState', pipeline_response)
+
+        if response.status_code == 202:
+            response_headers['Operation-Location']=self._deserialize('str', response.headers.get('Operation-Location'))
+            
+
+        if cls:
+            return cls(pipeline_response, deserialized, response_headers)
+
+        return deserialized
+
+
+
+    @distributed_trace
+    def begin_submit_conversation_job(
```

Comment:
I wonder if we should rename this operation to just `begin_conversation_analysis` or `begin_conversation_job`?
@heaths what do you think of this API name currently?

Reply:
I don't think we typically want to mention "job" or "task". It's typically that I've seen. Probably just `begin_conversation_analysis` as you suggested.

Reply:
renamed to `begin_conversation_analysis`
but it's a little confusing having
- sync operation as `analyze_conversation`
- async operation as `begin_conversation_analysis`

Reply:
That's a good point. I would bring it up with @johanste and @tg-msft.